backend/schema/corpora_schema.md

```diff
@@ -1,125 +1,68 @@
-
-
-### Software vs Policy
-
-Finally, we should draw a distinction between how we approach the schema and validation as a _policy_ versus how we implement it in _software_. The schema,
-especially anything outside of fundamental requirements, will change over time. It might change frequently and significantly as we get feedback from submitters
-and users. So while we might be strict about our validation policies at any given time, our software needs to be very flexible. We need to avoid a situation
-where we can't evolve schema and curation policies because it requires expensive software updates. There will be situations where software features or
-capabilities will be hard or impossible because we prioritize this flexibility.
+Then, when the submitter uploads the dataset to the Data Portal, the Data Portal will verify that the dataset does indeed follow an accepted version of the
+schema. If it does not, it will reject the dataset with an appropriate error message.
 
 ## Schema
 
-### Original Dataset Requirements
-
-
-These are the fundamental requirements that all dataset must meet, the handful of things our software depends on and mandatory legal requirements. All
-submitted data must comply with these requirements.
+### Basic Requirements
 
+There are a few requirements that are needed for the cellxgene Explorer to work correctly or are mandatory legal requirements:
 
 *   **Unique observation identifiers**. Each observation (usually a cell) must have an id that is unique within the dataset. This is usually already
     present in every submission, for example as a barcode.
 *   **Unique feature identifiers**. Every feature (usually a gene or transcript) also needs a unique identifier. This is occasionally not present because
-    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will be summed during
-    curation.
-*   **Type annotations or inferrable types**. For both expression values and metadata, we don't want to have to guess at the type. Typing is already explicit
-    in anndata, loom, Seurat and Bioconductor formats.
+    of one-to-many mappings between gene symbols and other gene ids. In cases where there are duplicated feature identifiers, they will need to be appropriately
+    combined before submission.
 *   **No PII**. No metadata can be personally identifiable: no names, dates of birth, specific locations, etc. There's a
-    [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing).
+    [list](https://docs.google.com/document/d/1nlUuRiQ8Awo_QsiVFi8OWdhGONjeaXE32z5GblVCfcI/edit?usp=sharing). It is not always possible to determine
+    automatically whether metadata contains PII. For example, a field that looks like a date could be a date of birth or just some other date. So tooling will
+    warn submitters for potential PII and allow them to attest that it is not personally identifiable.
```

Comment:
Identifying _potential PII_ is under discussion.

Reply:
:+1: I'll remove for now then